tests/test_api.py

```diff
@@ -41,14 +41,18 @@ def test_prefix_implicit_https(self):
         URL prefix given a host with no specified protocol.
         """
         with settings.runtime_values(host='33.33.33.33'):
-            self.assertEqual(client.prefix, 'https://33.33.33.33/api/v1/')
+            with mock.patch.object(Client, '_make_request') as req:
```

Comment:
Now prefix detection itself makes an HTTP request, so I had to mock it to return the expected JSON content from `/api/`.